docker/generic/Dockerfile

```diff
@@ -51,6 +51,9 @@ ENV PATH ${CONDA_PATH}/bin:$PATH
 
 # inject the renku-jupyter stack
 COPY --from=renku_base /opt/conda /opt/conda
+COPY --from=renku_base /usr/bin/tini /usr/bin/tini
+COPY --from=renku_base --chown=1000:1000 /usr/local/bin/*sh /usr/local/bin/
```

Comment:
what is this copying?

Reply:
This is copying the shell scripts that come in the base Jupyterlab image that are used to start the server. Amalthea needs these and they were not present before. I.e. Amalthea uses `start-notebook.sh` and the tests use the same command here.

Reply:
those are installed as part of the jupyterhub python package. If they are needed, we should probably just install jupyterhub. otoh, I'm not sure they _are_ needed. `start-notebook.sh` basically calls `start.sh` which does some checks for user permissions etc but can only correct them if running as root - since we don't run the containers as root I'm not sure it has any specific function. It also executes some hooks which I'm sure we don't use at the moment, but maybe users would expect that functionality? Not sure.

Reply:
Ok I remove these - now all the images just use the `jupyter lab ...` command to start.